### ISSUE TITLE
An error was encountered during the update process. Please try again.

### ISSUE THREAD

**Issue by ghost573** [Samsagax/gamescope-session#8]
Sorry, I'm new to Linux so I don't know to get logs for the display manager (?) to help but more than happy to provide them when shown how.

My device is the Aya Neo Founder (4500U with Vega iGPU) with the latest 5.18 kernel, I had issues with earlier builds but thought I try cb00bea to see if that fixes it but sadly not.

**Comment by Samsagax** [comment]
Hello. What are you trying to accomplish and what are the steps you did to reproduce? When and where did you get that message? Are you using ChimeraOS or some other Linux distro?

**Comment by ghost573** [comment]
>What are you trying to accomplish and what are the steps you did to reproduce?

Was hoping to have easy controls in games to toggle FSR, TDP etc without needing to remember the commands. I followed the readme and have files in the appropriate location.
```
/etc/systemd/system/gamescope@.service
/etc/pam.d/gamescope
/usr/bin/gamescope-session
```
I disabled my display manager, sddm and enabled gamescope however I get an error about cross linking.

>When and where did you get that message?

After selecting a network to connect to in the Steam Deck menu.

>Are you using ChimeraOS or some other Linux distro?

Sorry, forgot to mention. Vanilla arch with KDE Plasma as my DE. Kernel is up to date and is base-devel and amdgpu is up to date as well.
